packages/runtime/src/runtimes/react/hooks/use-selector.ts

```diff
@@ -1,4 +1,5 @@
-import { useSyncExternalStoreWithSelector } from 'use-sync-external-store/shim/with-selector'
+import { useSyncExternalStoreWithSelector } from 'use-sync-external-store/with-selector'
```

Comment:
React doesn't export `useSyncExternalStoreWithSelector`, so we have to keep this as a direct import from `use-sync-external-store` (but not the shim).